lib/pry-doc/pry_ext/method_info.rb

```diff
@@ -1,3 +1,5 @@
+require_relative "show_source_with_c_internals"
```

Comment:
Why do we use `require_relative` instead of traditional `require`? Check here, https://github.com/pry/pry-doc/blob/ddaeb5be6b8995e7e1d0ec68d4dc14fdbd4a50aa/lib/pry-doc.rb#L4

Maybe we could require from that place via `require`?

Reply:
@kyrylo because a full blown require for some of the files would be enormous, it would be like: `require 'pry-doc/pry_ext/show_source_with_c_internals/symbol_extractor'` etc